.gitignore

```diff
@@ -51,6 +51,8 @@ install/
 
 # Unignore the documentation build
 !isis/src/docsys/build
+!isis/src/docsys/Object/build
+!isis/src/docsys/Application/build
```

Comment:
Is this in preparation for splitting up the documentation builds?

Reply:
No, when I updated the conf files, git warned me that these were being ignore, and they should have been.